web/src/views/TwitterDownloaderView.vue

```diff
@@ -1,258 +1,249 @@
+									<div class="card-body">
+										<p class="card-text">
+											{{ tweetMetaData.description }}
+										</p>
+									</div>
 
-                            <div class="card-footer text-muted">
-                                <div class="row g-0 text-bg-dark ">
-                                    <div class="col-md-4 img-container">
-                                        <img
-                                            :src="tweetMetaData.thumbnail"
-                                            class="img-fluid rounded-start" />
-                                    </div>
+									<div class="card-body">
+										<ul>
+											<li
+												v-for="(item, index) in tweetMedias"
+												:key="index"   
+												type="button"                                                           
+												class="list-group-item thumbnail m-2 p-2 bg-primary rounded"
+												@click="downloadVideo(item.url)">
+												<i class="fa-solid fa-circle-arrow-down"></i>
 
-                                    <div class="col-md-8">
-                                        <div class="card-body">
-                                            <div class="card-text">
-                                                <table
-                                                    class="table table-dark table-hover">
-                                                    <thead>
-                                                        <tr>
-                                                            <th scope="col">
-                                                                Quality
-                                                            </th>
+												<span>
+													Download {{ item.resolution }} [{{ Number(index) + 1 }}]
+												</span>
+											</li>
+										</ul>
+									</div>
+                                    
+									<div class="card-footer">
+										<i class="fa-solid fa-video"></i>
+										{{ tweetMetaData.duration }}
+									</div>
+								</div>
+							</div>
+						</div>
+					</div>
+				</Transition>
+				
+				<div class="mb-3">
+					<!-- Form -->
+					<form
+						class="needs-validation"
+						@submit.prevent>
+						<div class="row rounded-top ">
+							<label class="form-label text-center">
+								Enter URL
+							</label>
+							
+							<div class="input-group">
+								<div class="input-group mb-3">
+									<TheInput
+										v-model="downloadForm.url"
+										type="text"
+										placeholder="Enter Video Link..."
+										:disabled="isLoading"
+										:class="{
+											'is-invalid': v$.url.$error && v$.url.$dirty,
+											'is-valid': !v$.url.$error && v$.url.$dirty
+										}"
+										@blur="v$.url.$touch" />
+										
+									<div
+										v-for="error of v$.url.$errors"
+										:key="error.$uid"
+										class="invalid-feedback">
+										{{ error.$message }}
+									</div>
+								</div>
+							</div>
```

Comment:
Why is TheInput needed to be in a 2 nested `input-group` class div? Do check this out as a reference https://github.com/pizza-society/unicornify/blob/PS-59-Refactor-Web/web/src/views/QRGeneratorView.vue#L28-L47